tests/ui/suggestions/option-content-move3.stderr

```diff
@@ -29,7 +37,59 @@ LL |             let x = var;
    |                     |
    |                     variable moved due to use in closure
    |                     move occurs because `var` has type `NotCopyable`, which does not implement the `Copy` trait
+   |
+note: if `NotCopyable` implemented `Clone`, you could clone the value
+  --> $DIR/option-content-move3.rs:2:1
+   |
+LL | struct NotCopyable;
+   | ^^^^^^^^^^^^^^^^^^ consider implementing `Clone` for this type
+...
+LL |             let x = var;
+   |                     --- you could clone this value
+
+error[E0507]: cannot move out of `var`, a captured variable in an `FnMut` closure
```

Comment:
Looks like the compiler is totally inconsistent about "a `FnMut`" vs "an `FnMut`" :/

Reply:
Which one should we use?

Reply:
I'll move towards `a FnMut` (in a separate PR) because making every place where we talk about trait names to be specialized based on the language rules of the actual *name* of the trait would be a headache, whereas moving in the other direction is just a handful of static strings.

Reply:
I just saw this in `compiler/rustc_ast/src/ast.rs`, which agrees with the use of "a":
```rust
impl ItemKind {
    pub fn article(&self) -> &'static str {
        use ItemKind::*;
        match self {
            Use(..) | Static(..) | Const(..) | Fn(..) | Mod(..) | GlobalAsm(..) | TyAlias(..)
            | Struct(..) | Union(..) | Trait(..) | TraitAlias(..) | MacroDef(..)
            | Delegation(..) => "a",
            ExternCrate(..) | ForeignMod(..) | MacCall(..) | Enum(..) | Impl { .. } => "an",
        }
    }
```